sync.pl

```diff
@@ -1,136 +1,301 @@
-#!/usr/bin/perl
+#!/usr/bin/env perl
+# sync.pl by azatoth (2011), update by amorymeltzer (2019)
 
-use v5.10;
+use strict;
+use warnings;
 
-use Git::Repository;
-use MediaWiki::Bot;
-use File::Slurp;
-use Getopt::Long::Descriptive;
-use Encode;
+use English qw(-no_match_vars);
 use utf8;
 
-my $config_file = "$ENV{HOME}/.mwbotrc";
-my $c = {do "$config_file"} if -f $config_file;
+use Config::General qw(ParseConfig);
```

Comment:
Excessive?  Maybe.  But the whole point of cpan is not having to reinvent the wheel, especially when it comes to loading files.